pandas/core/base.py

```diff
@@ -990,11 +990,17 @@ def factorize(self, sort=False, na_sentinel=-1):
         from pandas.core.algorithms import factorize
         return factorize(self, sort=sort, na_sentinel=na_sentinel)
 
-    def searchsorted(self, key, side='left'):
-        """ np.ndarray searchsorted compat """
+    def searchsorted(self, key, side='left', sorter=None):
```

Comment:
Actually, `DatetimeIndex.searchsorted` has no doc string at the moment.  See <a href="https://github.com/pydata/pandas/pull/12413/files?diff=unified#diff-23ecb29e7ceba52109a365e447400d2eL1630">here</a>